scriptmodules/supplementary/wifi.sh

```diff
@@ -26,6 +26,8 @@ function _set_interface_wifi() {
             ip link set wlan0 down
         fi
     fi
+    # signal wpa_supplicant to reconfigure the interface
+    wpa_cli -i wlan0 reconfigure 2>/dev/null
```

Comment:
This causes `OK` to be output and breaks wifi scanning for me. With this line I get

```wlan0     Interface doesn't support scanning : Device or resource busy```

It's run just before scanning so maybe it's in the wrong place ?

```
    _set_interface_wifi up 2>/dev/null (which includes the call)
    dialog --infobox "\nScanning for WiFi networks..." 5 40
```

Reply:
It's supposed to tell `wpa_supplicant` to re-read the conf file. 
It's probably better to move the call somewhere else, since this gets called before scanning and it's not supposed to. I'll take a look and change it.

Reply:
I've removed it and it looks like the `dhcpcd` restart takes care of the reloading. Probably that's reconfiguration  failed in my tests, since I removed the `dhcpcd` reloading.

Looks like there's still a hook run for `wpa_supplicant` in `/lib/dhcpcd/dhcpcd-run-hooks` that performs the reloading, so adding back the `dhchp` reload takes care of re-configuring `wpa_supplicant` after any config files are changed.